### PR TITLE
Fix Travis build

### PR DESCRIPTION
**Pull request by imurchie**
Update Travis config. Also move Azure build to use multiple reporters, and remove "build" stage in tests, since building happens in all the other stages.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -13,16 +13,20 @@ android:
...
     - _FORCE_LOGS=1
     - DEVICE=android
-    - MOCHA_TIMEOUT=360000
+    - MOCHA_TIMEOUT=900000
     - RECURSIVE=
     - START_EMU=1
     - ANDROID_API=21
     - ANDROID_EMU_NAME=test
-    - ANDROID_EMU_ABI=armeabi-v7a
+    - ANDROID_EMU_ABI=x86_64
```
Comment (mykola-mokhnach) [inline .travis.yml]: wow, does it work now on their pods?

File: .travis.yml [hunk 2]
```diff
@@ -39,19 +43,20 @@ before_install:
...
-      for apiVersion in 15 16 17 18 19; do
-        sdkmanager --uninstall "system-images;android-${apiVersion};default;armeabi-v7a" > /dev/null
+      for apiVersion in 18 19; do
         sdkmanager --uninstall "platforms;android-${apiVersion}" > /dev/null
       done
-      echo yes | sdkmanager --update > /dev/null
-      echo yes | sdkmanager "platforms;${ANDROID_EMU_TARGET}" > /dev/null
-      echo yes | sdkmanager "extras;android;m2repository" > /dev/null
-      # echo yes | sdkmanager "extras;google;m2repository" > /dev/null
+      echo y | sdkmanager "build-tools;28.0.3" >/dev/null
```
Comment (mykola-mokhnach) [inline .travis.yml]: Can we avoid hardcoding version numbers?

File: .travis.yml [hunk 3]
```diff
@@ -39,19 +43,20 @@ before_install:
...
+      echo y | sdkmanager --update > /dev/null
+      echo y | sdkmanager "platforms;${ANDROID_EMU_TARGET}" > /dev/null
+      echo y | sdkmanager "platforms;android-28" >/dev/null
```
Comment (mykola-mokhnach) [inline .travis.yml]: same here

File: .travis.yml [hunk 4]
```diff
@@ -39,19 +43,20 @@ before_install:
...
+      echo y | sdkmanager "extras;android;m2repository" > /dev/null
+      echo y | sdkmanager --channel=4 "emulator" >/dev/null
```
Comment (mykola-mokhnach) [inline .travis.yml]: what is channel4?

https://developer.android.com/studio/command-line/sdkmanager says it should be in range 0-3

Comment (imurchie) [reply .travis.yml]: Specifies Canary.

Comment (mykola-mokhnach) [reply .travis.yml]: > --channel=channel_id | Include packages in channels up to channel_id. Available channels are:0 (Stable), 1 (Beta), 2(Dev), and 3 (Canary).

Perhaps my tutorial is different than yours %)

**Pull request merged**